report/renderer/util.js

```diff
@@ -431,9 +431,9 @@ class Util {
       case 'devtools': {
         const {cpuSlowdownMultiplier, requestLatencyMs} = throttling;
         cpuThrottling = `${Util.i18n.formatNumber(cpuSlowdownMultiplier)}x slowdown (DevTools)`;
```

Comment:
probably `formatInteger` on the cpu throttling as well?

Reply:
While we don't ever assign this value to non-integer numbers, others might via the CLI.

Reply:
fair point. `2.0x` looks kind of dumb, though :)

Reply:
`formatNumber` could take a param for sig figs?

Reply:
hold up why don't we just do `${cpuSlowdownMultiplier}x`

this entire string gets a F for i18n atm anyway so we still need to do this correctly with a ICU message

Reply:
oh right, arabic